lib/models/messages/twitch/event.dart

```diff
@@ -100,7 +106,9 @@ class TwitchPollEventModel extends MessageModel {
         pollTitle: data['event']['title'],
         isCompleted: true,
         messageId: "poll${data['event']['id']}",
-        timestamp: data['timestamp'].toDate());
+        timestamp: data['timestamp'].toDate(),
+        startTimestamp: data['event']['started _at'].toDate(),
+        endTimestamp: data['event']['ended_at'].toDate());
```

Comment:
I guess the only thing that we might want to test with some data is if pin behaviour should change depending on the `status` property, maybe we don't want to pin in on of the two other status (archived and/or terminated)

Reply:
I will make it auto unpin when the poll status is `archived` or `terminated`.